### DISCUSSION TITLE
Let's chat about ...

### DISCUSSION THREAD

**Discussion by sbueringer** [kubernetes-sigs/cluster-api#6106] [General]
Hey folks,

we are now hosting "Let's chat about ..." sessions. The idea is to come together as a community and talk about topics relevant to contributors and users of the project. We want to make them as interactive as possible and not just "simple" talks about specific topics. 

We are starting with the following sessions:

| Date              | Topic                                        | Meeting     | Recording |
| ----------------- | -------------------------------------------- | ----------- | --------- | 
| 2022-02-10 11 AM CET | Code structure & Makefile targets (APAC/EMEA)  | Done   | [YouTube](https://www.youtube.com/watch?v=Y6Gws65H1tE) |
| 2022-02-14 07 PM CET  | Code structure & Makefile targets (EMEA/Americas)  | Done  | [YouTube](https://www.youtube.com/watch?v=_prbOnziCJw) |
| 2022-02-24 10 AM CET  | Local CAPI development and debugging with Tilt (APAC/EMEA)  | Done   | [YouTube](https://www.youtube.com/watch?v=CM-dotO2nSU) |
| 2022-02-28 07 PM CET  | Local CAPI development and debugging with Tilt (EMEA/Americas)  | Done  | [YouTube](https://www.youtube.com/watch?v=tEIRGmJahWs)  |
| TBD  | Build, E2E tests & test framework (EMEA/APAC)  | TBD   |  |
| TBD  | Build, E2E tests & test framework (EMEA/Americas)  | TBD   |  |

**Calendar**: [Cluster API - Let's chat about ...](https://calendar.google.com/calendar/embed?src=1s7pv1koij1g0vek2c9pit8ruk%40group.calendar.google.com)

I will update this discussion continuously, so if you're interested, please subscribe and join the sessions :)

If you have any questions about those topics, please bring them with you and ask at any time!

If you have ideas for topics, the format in general, feedback, ... feel free to discuss below.

Similar series:
* [CAPA Code Walkthroughs](https://github.com/kubernetes-sigs/cluster-api-provider-aws/discussions/3266)

Topic backlog:
* ClusterClass code walkthrough

**Comment by vincepri** [comment]
Can we also collect these in the book for easy finding?

**Reply by fabriziopandini** [reply]
We just added in the book a link to this thread, so we don't have to open a PR for each session 😉

**Reply by vincepri** [reply]
That's fine for now, we should consider to have the content in one place to avoid having it spread out

**Reply by sbueringer** [reply]
Link is here in the book: https://cluster-api.sigs.k8s.io/developer/guide.html#videos-explaining-capi-architecture-and-code-walkthroughs

I think it's a similar situation with the CAPI Office hour videos. There are frequently new ones and we add them to the YouTube playlist, but we don't want to open weekly PRs to add them to the book.

(no objections to adding a new batch once in a while, but I think let's first see how good this format works)

**Reply by vincepri** [reply]
Yep, batching once in a while sounds good!

**Comment by sbueringer** [comment]
Changelog:

Scheduled next session for 24th Februrary 10 AM CET

**Reply by sbueringer** [reply]
Scheduled next session for 28th Februrary 07 PM CET

**Reply by sbueringer** [reply]
Uploaded recording for `Local CAPI development and debugging with Tilt (EMEA/APAC)`

**Reply by sbueringer** [reply]
Uploaded recording for `Local CAPI development and debugging with Tilt (EMEA/Americas)`

**Reply by sbueringer** [reply]
Added link to ongoing CAPA code walkthrough series. Highly recommended!

@sedefsavas and @richardcase Thank you very much for organizing/hosting it!

**Comment by fabriziopandini** [comment]
I'm closing this because it is some time now that we don't get new requests, and we also did other focused sessions e.g. withing the work of the CI release team.

All the videos are linked from the CAPI book, and we can always hold new sessions if a specific ask pops up
